addons/survey/views/survey_conditional_end_message_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="survey_conditional_end_message_form" model="ir.ui.view">
```

Comment:
I'm still not in favor of this view. Have you considered adding the field to the embedded list of questions directly, wouldn't it be more convenient?

Reply:
keeping the view until PO decision. Adding a extra messages column to the survey question list.